ci-operator/config/openshift/openshift-state-metrics/openshift-openshift-state-metrics-release-4.1.yaml

```diff
@@ -40,6 +40,10 @@ tests:
   commands: make test-unit
   container:
     from: src
+- as: e2e-aws
```

Comment:
I dunno if we need to go all the way back to 4.1 ;).  But I guess it doesn't hurt either...

Reply:
I went back and forth on this and then went with the safer option :-)